### DISCUSSION TITLE
Forking is not Working. Tried different WIFI and accounts.

### DISCUSSION THREAD

**Discussion by saintramon** [community/community#47137] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Tried forking 2 repositories. None of them worked.

I switched accounts to determine if my account is the problem, still doesn't work. Switched WIFI because maybe my connection is the problem, still forking doesn't work.

**Comment by zafergurel** [comment]
Same here!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @saintramon ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by queenofcorgis** [comment, marked as answer]
This is now resolved - as we are not seeing these errors show up in our monitoring. Our monitors show this event happened February 12 between 6-8am ET. However, if you are still seeing this persist or have had this occur more recently, please let me know ❤️ .
